notebooks/hello-npu/hello-npu.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    ONNX_CV_MODEL_PATH = MODEL_DIRECTORY_PATH / f'{model_name}.onnx'
</pre><p>does it really required to convert model to onnx? is it not possible to use pytorch frontend directly? (if you use onnx only for avoiding dynamic shapes, you can pass input parameter with required shape into ov.convert_model.</p><p>Please check <a href="https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/pytorch-to-openvino/pytorch-to-openvino.ipynb" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/pytorch-to-openvino/pytorch-to-openvino.ipynb</a></p><p>for more examples</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Alright, makes sense. Changed!